### PR TITLE
Feature/issue 103/refactor add skill level field

### PR DESCRIPTION
**Pull request by DanDuguay**
This PR is to complete issue #103.

## Changes

- Added a skillLevelEnum with values "BEGINNER", "INTERMEDIATE", and "ADVANCED" to be used when creating/joining events.
- Added Unit tests

### CODE REVIEW

**Review by Ziadsharkos — DISMISSED** [review]
Good additions, great work!

File: Microservices/event-service/src/main/java/app/sportahub/eventservice/dto/response/EventResponse.java [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 import app.sportahub.eventservice.model.event.Location;
 import app.sportahub.eventservice.model.event.Participant;
 import app.sportahub.eventservice.model.event.Team;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import java.time.LocalDate;
+import java.util.EnumSet;
 import java.util.List;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public record EventResponse(String id, String eventName, String eventType, String sportType, Location location,
```
Comment (walidoow) [inline Microservices/event-service/src/main/java/app/sportahub/eventservice/dto/response/EventResponse.java:13]: Can you replace the Location object in response with LocationResponse (you will need to create it).

**Review by walidoow — CHANGES_REQUESTED** [review]

**Comment by walidoow** [comment]
@DanDuguay you could have named it location directly instead of locationRequest.
image

That way you would not have to specifically map the fields in the mapper.
image

**Review by walidoow — APPROVED** [review]
Looks good, great job 👍

**Review by Ziadsharkos — APPROVED** [review]

**Comment by walidoow** [comment]
@DanDuguay watch out you have duplicated commits

**Review by walidoow — APPROVED** [review]

**Review by Ziadsharkos — APPROVED** [review]

**Pull request merged**